src/main/java/pingis/config/SecurityDevConfig.java

```diff
@@ -23,7 +23,7 @@ public void configure(WebSecurity web) throws Exception {
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        http
+        http.csrf().ignoringAntMatchers("**").and()
```

Comment:
Done as per the instructions (the REST endpoint solution specifically) at
https://docs.spring.io/spring-security/site/docs/current/reference/html/websocket.html#websocket-sameorigin-csrf
https://docs.spring.io/spring-security/site/docs/current/reference/html/websocket.html#websocket-sockjs-csrf